## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 474629`
Commit: [`625861e97f485447a3a16b6a32c77fc330af2871`](https://github.com/NixOS/nixpkgs/commit/625861e97f485447a3a16b6a32c77fc330af2871) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/625861e97f485447a3a16b6a32c77fc330af2871..pull/474629/head))
Merge: [`7c11960bf337268f04253e4878056f1c97279dcf`](https://github.com/NixOS/nixpkgs/commit/7c11960bf337268f04253e4878056f1c97279dcf)

Logs: https://github.com/sikmir/nixpkgs-review-gha/actions/runs/20579078034


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>orbiton</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>orbiton</li>
  </ul>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>orbiton</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>orbiton</summary>
<pre>Running phase: unpackPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;unpackPhase&quot; }
unpacking source archive /nix/store/k0m9lk7j5ixii3jnpaph4i2r52vdzxp3-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Building subPackage .
buildPhase completed in 1 minutes 11 seconds
Running phase: checkPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;checkPhase&quot; }
--- FAIL: TestPBcopy (0.00s)
FAIL

FAIL	github.com/xyproto/orbiton/v2	5.456s
FAIL</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>orbiton</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>orbiton</summary>
<pre>Running phase: unpackPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;unpackPhase&quot; }
unpacking source archive /nix/store/k0m9lk7j5ixii3jnpaph4i2r52vdzxp3-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Building subPackage .
Running phase: checkPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;checkPhase&quot; }
--- FAIL: TestPBcopy (0.00s)
FAIL

FAIL	github.com/xyproto/orbiton/v2	1.900s
FAIL</pre>
</details>
</details>

--- a/pkgs/by-name/or/orbiton/package.nix
+++ b/pkgs/by-name/or/orbiton/package.nix
@@ -12,13 +12,13 @@
 
 buildGoModule rec {
   pname = "orbiton";
-  version = "2.70.0";
+  version = "2.70.4";
 
   src = fetchFromGitHub {
     owner = "xyproto";
     repo = "orbiton";
     tag = "v${version}";
-    hash = "sha256-3EAYPCNVQiED8qHyLbwyYU7gXJn2TFgiJ2/JyxyD7+M=";
+    hash = "sha256-cdaYD6PyOjgBo83eWD2+YWQj5uJzmeHDo67dlA7Pj1g=";
   };
 
   vendorHash = null;
@@ -33,9 +33,14 @@ buildGoModule rec {
 
   preBuild = "cd v2";
 
-  checkFlags = [
-    "-skip=TestPBcopy" # Requires impure pbcopy and pbpaste
-  ];
+  checkFlags =
+    let
+      skippedTests = [
+        "TestPBcopy" # Requires impure pbcopy and pbpaste
+        "TestPkill" # error: no process named "sleep" found
+      ];
+    in
+    [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];
 
   postInstall = ''
     cd ..

